### ISSUE TITLE
Unknown: Failed to get convolution algorithm. This is probably because cuDNN failed to initializ

### ISSUE THREAD

**Issue by lw3259111** [deepmipt/DeepPavlov#1150]
python =3.7.3
tensorflow-gpu=1.15.0
cuda=10.0

when i use `KerasClassificationModel` to train classifier model, the model is error with
`` `
Unknown: Failed to get convolution algorithm. This is probably because cuDNN failed to initializer
```
but i use 
```
mnist = tf.keras.datasets.mnist

(x_train, y_train), (x_test, y_test) = mnist.load_data()
x_train, x_test = x_train / 255.0, x_test / 255.0
model = tf.keras.models.Sequential([
  tf.keras.layers.Flatten(input_shape=(28, 28)),
  tf.keras.layers.Dense(128, activation='relu'),
  tf.keras.layers.Dropout(0.2),
  tf.keras.layers.Dense(10, activation='softmax')
])

model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy'])
model.fit(x_train, y_train, epochs=5)

model.evaluate(x_test,  y_test, verbose=2)
```
the code is fine.
so How can i do?

**Comment by dilyararimovna** [comment]
Hi! 
You probably need to downgrade cuda to cuda9 or update tf.
Ref: [issue in tf repo](https://github.com/tensorflow/tensorflow/issues/24828)

**Comment by yoptar** [comment]
Hi @lw3259111,
Do you have cudnn installed? What version?
Could you also show us the whole output log with this error?

**Comment by jude90** [comment]
check you GPU resouces usage by  command `nvidia-smi` . It happens when memory usage is overload

**Comment by Jamesbond32** [comment]
I am getting the same error after installing tensorflow-gpu with conda. I ran nvidia-smi and got the following results. 

`+-----------------------------------------------------------------------------+
| NVIDIA-SMI 440.64       Driver Version: 440.64       CUDA Version: 10.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce RTX 207...  Off  | 00000000:01:00.0 Off |                  N/A |
| N/A   44C    P5    14W /  N/A |   7962MiB /  7982MiB |      5%      Default |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    0      2047      G   /usr/lib/xorg/Xorg                           149MiB |
|    0      2176      G   /usr/bin/gnome-shell                         172MiB |
|    0      2650      G   ...AAAAAAAAAAAAAAgAAAAAAAAA --shared-files    76MiB |
|    0     22115      C   ...skye/anaconda3/envs/keras_VE/bin/python  7557MiB |
+-----------------------------------------------------------------------------+
(keras_VE) skye@system-76:~$ 

` 

im not sure what to make of this print out. This algorithm used to run on my CPU just fine. is this saying that im using 7982 out of 7962Mib? what can I do to correct this? I have a GForce RTX 2070 which should be able to handle 10,000 images without an issue.
